delta/sdk/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/PermissionsSpec.scala

```diff
@@ -0,0 +1,149 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.model
```

Comment:
Tests are way too compacted, I cannot understand much. Wouldn't it be easier to separate it in several scalatest blocks?

Reply:
It's a straightforward (state, event) => state and (state, command) => event test series that cover all cases. I could add more comments if that helps to show what's tested.

Reply:
Is the same to add comments as `//` than scalatest blocks. 

They are there to group tests and give some extra context, otherwise we would always do that, and clutter everything in one.

But in anyway is done is fine :)